src/Parser/CommandInfo.php

```diff
@@ -334,6 +339,25 @@ public function getTopics()
     }
 
     /**
+     * Returns an array of commands that this command calls.
+     *
+     * @return string[]
+     */
+    public function getCalls()
+    {
+        return $this->calls;
+    }
+
+    public function setCalls($calls) {
+        if (is_string($calls)) {
+            $calls = explode(',', static::convertListToCommaSeparated($calls));
+        }
+        $this->calls = array_filter($calls);
```

Comment:
Yes I'd prefer that. I briefly attempted this but wasn't quite sure how to access all `@calls` values, each usage appeared to overwrite the previous.